benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/tasks.py

```diff
@@ -371,11 +376,11 @@ def intercept(
         # chunk sent back is used to calculate time to first token(TTFT).
         for response in responses:
             if ttft == 0:
-                ttft = time.perf_counter() - start_perf_counter
+                ttft = (time.perf_counter() - start_perf_counter) * 1000
```

Comment:
Checking my own understanding of this :) Is this to convert ttft to ms? assuming this is a bug fix since we did the *1000 in the comments but not the actual var

Reply:
Yeah, the data is still technically accurate. But definitely inconsistent with the total response time, and a bit weird to reason about as a decimal

Reply:
sg ty for confirming!